10981.0.txt

```diff
@@ -0,0 +1,36 @@
+@0
+
+
+@1
+MOV UP, RIGHT
+
+
+@2
+# Add the number to the accumulator.
+ADD LEFT
```

Comment:
Could just do `MOV LEFT, ACC` here -- then you also wouldn't need to zero the accumulator at the end